sympy/utilities/lambdify.py

```diff
@@ -50,9 +50,9 @@
     "E": "e",
     "I": "j",
     "ln": "log",
-    #"lowergamma":"lower_gamma",
+    "lowergamma":"gammainc",
```

Comment:
Since `lowergamma` and `uppergamma` are different functions, I don't think they should have the same translation `gammainc`.